components/cloud-foundry/frontend/test/unit/view/applications/application/summary/add-route.service.spec.js

```diff
@@ -172,8 +173,151 @@
       };
 
       data.useRandomPort = true;
+      data.activeTab = 0;
+      modalObj.actionTask(data, dialog);
+      $httpBackend.flush();
+    });
+
+    it('should successfully add an existing route', function () {
+
+      var routesRequestUrl = '/pp/v1/proxy/v2/spaces/' +
+          spaceGuid +
+          '/routes?include-relations=domain,apps&inline-relations-depth=1&results-per-page=100';
+
+      var expectedRoutesRes = {
```

Comment:
There's an old practise of placing large responses in *.mock.spec.js files. Think the original plan was to build up a library of calls. Here's it's ok but if there's multiple it may be best to move to route.model.spec.js